packages/cactus-test-tooling/package.json

```diff
@@ -101,6 +101,7 @@
     "@types/fs-extra": "9.0.13",
     "@types/js-yaml": "4.0.3",
     "@types/lodash": "4.14.172",
+    "@types/node": "20.11.24",
```

Comment:
Can we use types for node V18 (minimal supported version) instead?

Reply:
@outSH Oops, yes, of course! Sorry for accidentally trying to sneak Node v20 in again... :-)